spec/system/users/manual_sign_out_spec.rb

```diff
@@ -19,7 +19,7 @@
       visit edit_user_path
 
       # User should be signed out.
-      expect(page).to have_text('Continue with email')
+      expect(page).to have_text(user.school.name)
```

Comment:
This is not a definitive check for the user being on the home-page. It might be straight-forward to simply check for a known unique id / class `have_selector` in this case.